### PR TITLE
Tested TravisCI build. Build exited with 0.(Success-Level 4)

### PR DESCRIPTION
**Pull request by ManrajGrover**
Hi!

**Do not Merge**

This is just a test branch for successful build of the branch.(Level 4 complete)
ss 2015-04-10 at 08 16 25

### CODE REVIEW

**Comment by smcgregor** [comment]
OK, keep in mind that when you push commits for a PR, tests will run twice. Once on your TravisCI account and once on the Privly Foundation's TravisCI account. For security reasons most of the tests don't run when they are run on the Foundation's account through a PR.

**Comment by ManrajGrover** [comment]
Oh okay sir! I will test it first on forked repository before opening any further PR. :)

I am currently facing some issues in setting up Level 3. I will try again to fix it. If not, I will open an issue for discussing the fix.

File: shared/test/local_storage.js [hunk 1]
```diff
@@ -37,5 +37,6 @@ describe ("Local Storage Test Suite", function() {
     expect(returned["bar"]).toBe("bazz");
     expect(returned).toEqual(obj);
   });
-
+  // Adding a random comment for testing TravisCI build
```
Comment (smcgregor) [inline shared/test/local_storage.js]: You should not include these changes.

File: .travis.yml [hunk 2]
```diff
@@ -26,6 +26,9 @@ env:
   - secure: CevDxddWV/R2vF66U94Id30vSkQ2I1DBY36bl2VPyY5OsBGhSFWaF0l5j6KIgBZyYrwQF2IT1I/hUPtLbTvHhPgaE5CcV1LHGxueWtaYEXfAfHSVy0ChhhilKdUOOyA7245yad6UHYQIwx7Kz/EKXXA/0hczEcUl/OH6ZxkGcws=
   #CODECLIMATE_REPO_TOKEN=**************************(privly/privly-applications)
   - secure: HAqHN7zeSbDR7BP0pn6qH6ODOwuMD7eIR9t4h1gHkRy6WAQBrhQb1+kTzkGbhfKB1mftxN/0ERTK6xv+iBj4QylqOOXChnuthuHMZiAuwPJncj8FYE4LHt2Eh48+HnzSQUgzjxv11hvkLz4utaj0J4fub2q/tWzoAXucA13UGQU=
+  - secure: O0Su8HvBR3/4hUu9aBqD+TyBqPnZWGwHmozlDR6NLLSwXZAIJ/7dNjmfaiJonaR6UX2oNTk91uL0yJgCZ8lfbDjqtcNTUTwfOfQawRG3RNow72G7fPRaN3CkWP8oJlF2vHzz6263C3TUEt+BGE/CAPAyF48tTHp8teo0foCTGr0=
```
Comment (smcgregor) [inline .travis.yml]: You need to add a comment so we will know whose variable this is.

**Comment by smcgregor** [comment]
Whoops, I should not have reviewed this. I am closing this since I don't think you mean for this to be merged.

**Comment by ManrajGrover** [comment]
Yes sir. I don't think it is required to be merged, or is it?

**Comment by smcgregor** [comment]
Nope.

**Pull request closed without merging**